src/message.coffee

```diff
@@ -131,26 +134,43 @@ class SlackTextMessage extends TextMessage
   ###*
   # Returns name of user with id
   ###
-  replaceUser: (id, client) ->
-    return client.web.users.info(id).then((user) -> "@#{user.name}")
+  replaceUser: (client, id, mentions) ->
+    client.web.users.info(id).then((user) ->
+      if user
+        mention = new SlackMention(user.id, 'user', user)
+        mentions.push(mention)
+        return "@#{user.name}"
+      else return link
+    )
+    .catch((error) =>
```

Comment:
if this error happens, what impact will there be on the result of `replaceLinks()`?

Reply:
Good call. I modified the catch to return back the original link (`<@Uxxx>` or `<#Cxxx>`) which I would assume is desired behavior.